.github/workflows/normalize-titles.yml

```
@@ -0,0 +1,148 @@
+name: Normalize Issue And PR Titles
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)